internal/ingestion/knowledge_compile/wiki_map_store.go

```
@@ -0,0 +1,179 @@
+// discriminator, while preserving every chunk/hash version for reuse.
+func NewWikiMapVersionStore(docEngine engine.DocEngine) kccommon.WikiMapVersionStore {
+	return &wikiMapVersionStore{engine: docEngine}
+}
+
+func (s *wikiMapVersionStore) GetWikiMapVersions(ctx context.Context, tenantID, datasetID string, keys []string) (map[string][]byte, error) {
+	out := make(map[string][]byte, len(keys))
+	if len(keys) == 0 {
+		return out, nil
+	}
+	if s == nil || s.engine == nil {
+		return nil, fmt.Errorf("Wiki MAP version DocStore is not initialized")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)